nullaway/src/main/java/com/uber/nullaway/ErrorProneCLIFlagsConfig.java

```diff
@@ -165,6 +167,14 @@
     jarInferRegexStripModelJarName = flags.get(FL_JI_REGEX_MODEL_PATH).orElse(BASENAME_REGEX);
     jarInferRegexStripCodeJarName = flags.get(FL_JI_REGEX_CODE_PATH).orElse(BASENAME_REGEX);
     errorURL = flags.get(FL_ERROR_URL).orElse(DEFAULT_URL);
+    if (acknowledgeAndroidRecent && !isAcknowledgeRestrictive) {
+      throw new IllegalStateException(
+          "-XepOpt"
```

Comment:
"-XepOpt:NullAway" ;)

Reply:
The values of the `FL_*` variables all start with `NullAway` I think.  But I will add the missing colon 😄